src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -173,7 +173,7 @@ class GordonSchedulePanel extends Component {
       THU_CDE: mySchedule.thursday ? 'R' : null,
       FRI_CDE: mySchedule.friday ? 'F' : null,
       SAT_CDE: mySchedule.saturday ? 'S' : null,
-      SUN_CDE: mySchedule.sunday ? 'N' : null,
+      // SUN_CDE: mySchedule.sunday ? 'N' : null,
```

Comment:
```suggestion
      // SUN_CDE: mySchedule.sunday ? 'N' : null,
```

remove if unused

Reply:
I merged develop into this branch to update our protected branch with the develop changes, which is where a lot of this came from...